ts_scripts/install_dependencies.py

```diff
@@ -175,20 +177,11 @@ def get_brew_version():
 
 if __name__ == "__main__":
     check_python_version()
-    parser = argparse.ArgumentParser(
-        description="Install various build and test dependencies of TorchServe")
-    parser.add_argument('--cuda',
-                        default=None,
-                        choices=['cu92', 'cu101', 'cu102', 'cu111', 'cu113'],
-                        help="CUDA version for torch")
-    parser.add_argument(
-        '--environment',
-        default='prod',
-        choices=['prod', 'dev'],
-        help=
-        "environment(production or developer) on which dependencies will be installed"
-    )
-
+    parser = argparse.ArgumentParser(description="Install various build and test dependencies of TorchServe")
+    parser.add_argument('--cuda', default=None, choices=['cu92', 'cu101', 'cu102', 'cu111'], help="CUDA version for torch")
+    parser.add_argument('--environment', default='prod', choices=['prod', 'dev'],
+                        help="environment(production or developer) on which dependencies will be installed")
+    parser.add_argument("--reinstall_dependencies", default='no', choices=['yes', 'no'], help="force reinstall dependencies")
```

Comment:
Since this is a boolean value, use action="store_true", instead of specifying 'yes', 'no'.